packages/rum-core/src/performance-monitoring/performance-monitoring.js

```diff
@@ -266,9 +266,9 @@ export default class PerformanceMonitoring {
         })
       }
 
-      const span = transactionService.startSpan(spanName, 'external.http')
-      const taskId = transactionService.addTask()
-
+      const span = transactionService.startSpan(spanName, 'external.http', {
+        blocked: true
```

Comment:
might change it to `task` as it's confusing with `sync` flag.

Reply:
or we can keep it to `blocking`.

Reply:
I think using `withTask` might be a better option.

Reply:
Also, let's make sure that this option is logged in the debug mode. This helps us debug if a transaction never ends.

Reply:
Thinking about it more, we want to stick to few concepts as possible and with this option setting to `task` or `withTask`, seems like we are teaching yet another concept to user. Lets think of a way to hide the task concept itself in general and provide another way to deal with it.